src/axom/slic/core/Logger.hpp

```diff
@@ -173,6 +173,13 @@ class Logger
    * \param [in] filter_duplicates optional parameter that indicates whether
    * duplicate messages resulting from running in parallel will be filtered out.
    * Default is false.
+   * \note When used within an MPI distributed environment, and:
```

Comment:
@bmhan12 since the comments you added to each of these methods is essentially the same, I think it would be more effective/impactful if the comment appeared once in a "prologue" and state that it applies to each of these methods. Specifically, use method grouping syntax in Doxygen like we do in SIdre, for example: https://github.com/LLNL/axom/blob/develop/src/axom/sidre/core/Group.hpp#L129

Also, I would move the part about the methods being collective operations and that all ranks must call them to the front before the qualifications based on message level.